### PR TITLE
chore: say goodbye to bendctl

### LINKED ISSUE #4426: Consider remove bendctl
### Discussed in https://github.com/datafuselabs/databend/discussions/4423

<div type='discussions-op-text'>

<sup>Originally posted by **BohuTANG** March 13, 2022</sup>
bendctl is no used now, and we recommend that use our guideline to deploy Databend:
https://databend.rs/doc/category/deploy

We need redesign the new `bendctl` in the future to meet the needs of on-perm deployment.

cc @ZhiHanZ </div>

### PR DESCRIPTION
**Pull request by PsiACE**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

- remove bendctl
- clean up traces of bendctl

Please check the docker workflow and makefile in particular, in case I missed something.

## Changelog

- Build/Testing/CI

## Related Issues

Fixes #4426

## Test Plan

Unit Tests

Stateless Tests

### CODE REVIEW

File: .github/workflows/databend-release.yml [hunk 1]
```diff
@@ -169,6 +146,17 @@ jobs:
...
+      - name: Build and push
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          push: true
+          tags: ${{ secrets.DOCKERHUB_NAMESPACE }}/databend:${{ steps.get_version.outputs.VERSION }}
+          platforms: linux/amd64
+          context: .
+          file: ./docker/Dockerfile
+          build-args: VERSION=${{ steps.get_version.outputs.VERSION }}
```
Comment (PsiACE) [inline .github/workflows/databend-release.yml:158]:
```suggestion
          file: ./docker/Dockerfile
```

I think it has the same functionality as `file: ./docker/release/Dockerfile`. However, the `build-args` should be removed, as it is useless.

File: .github/workflows/databend-release.yml [hunk 2]
```diff
@@ -142,24 +128,15 @@ jobs:
...
-
       - name: Download binaries for usage
         id: download_binaries
         run: |
           wget https://repo.databend.rs/databend/${{ steps.get_version.outputs.VERSION }}/databend-${{ steps.get_version.outputs.VERSION }}-x86_64-unknown-linux-gnu.tar.gz
           wget https://repo.databend.rs/databend/${{ steps.get_version.outputs.VERSION }}/databend-${{ steps.get_version.outputs.VERSION }}-aarch64-unknown-linux-gnu.tar.gz
+          mkdir -p ./target/release
           mkdir -p ./target/x86_64-unknown-linux-gnu/release
           mkdir -p ./target/aarch64-unknown-linux-gnu/release
+          tar xC ./target/release -f ./databend-${{ steps.get_version.outputs.VERSION }}-x86_64-unknown-linux-gnu.tar.gz
```
Comment (PsiACE) [inline .github/workflows/databend-release.yml:139]: It will help us to use  `file: ./docker/Dockerfile` to publish the image

File: Makefile [hunk 3]
```diff
@@ -118,16 +87,6 @@ management-test: build-debug
...
 
-stateful-ctl-test:
-	rm -rf ./_meta*/
-	rm -rf ./.databend/
-	ulimit -n 10000;ulimit -s 16384; bash ./scripts/ci/ci-run-stateful-tests-standalone-ctl.sh
-
-stateful-ctl-cluster-test:
-	rm -rf ./_meta*/
-	rm -rf ./.databend/
-	bash ./scripts/ci/ci-run-stateful-tests-cluster-ctl.sh
```
Comment (PsiACE) [inline Makefile:129]: Still need to consider adding a handy command to run stateful tests.

File: Makefile [hunk 4]
```diff
@@ -147,9 +106,6 @@ docker:
 k8s-docker:
 	bash ./scripts/build/build-k8s-runner.sh
 
-docker_release:
-	docker buildx build . -f ./docker/release/Dockerfile  --platform ${PLATFORM} --allow network.host --builder host -t ${HUB}/databend:${TAG} --build-arg VERSION=${VERSION} --push
```
Comment (PsiACE) [inline Makefile:151]:
```shell
docker_release:
	docker buildx build . -f ./docker/Dockerfile  --platform ${PLATFORM} --allow network.host --builder host -t ${HUB}/databend:${TAG} --build-arg VERSION=${VERSION} --push
```

Maybe it should be changed to this?

File: docs/index.md [hunk 5]
```diff
@@ -1,112 +0,0 @@
----
-title: Get Started
```
Comment (ZhiHanZ) [inline docs/index.md:2]: why we delete index doc page?

File: .github/workflows/databend-release.yml [hunk 6]
```diff
@@ -169,6 +146,17 @@ jobs:
...
 
+      - name: Build and push
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          push: true
+          tags: ${{ secrets.DOCKERHUB_NAMESPACE }}/databend:${{ steps.get_version.outputs.VERSION }}
+          platforms: linux/amd64
+          context: .
+          file: ./docker/Dockerfile
```
Comment (ZhiHanZ) [inline .github/workflows/databend-release.yml:157]: not sure about whether the dockerfile currently works, we need to update on configuration changes...

File: docs/index.md (hunk 5 above)
Comment (PsiACE) [reply docs/index.md:2]: This document is outdated and we no longer link to it.

**Comment by Xuanwo** [comment]
Nice work! I love the idea to remove bendctl at this stage.

**Review by Xuanwo — APPROVED** [review]

**Comment by Xuanwo** [comment]
BTW, I felt so bored after writing a lot of `databend-query` and `databend-meta`. Is it a good idea to rename them to `bendquery` and `bendmeta`?

**Comment by PsiACE** [comment]
> BTW, I felt so bored after writing a lot of `databend-query` and `databend-meta`. Is it a good idea to rename them to `bendquery` and `bendmeta`?

I like the current style, which has a strong correlation to databend. (emmm, for example: cargo & cargo-audit， but we are not yet integrated together.).

**Comment by BohuTANG** [comment]
@mergify update

**Pull request merged**